Formula/netlify-cli.rb

```diff
@@ -28,9 +28,9 @@ def install
 
   test do
     (testpath/"test.exp").write <<~EOS
-      spawn #{bin}/netlify login
+      spawn #{bin}/netlify help
       expect "Opening"
     EOS
-    assert_match "Logging in", shell_output("expect -f test.exp")
+    assert_match "COMMANDS", shell_output("expect -f test.exp")
```

Comment:
We don't want to test help commands since they rarely reveal problems. Maybe we can remove the `expect` call to reveal the issue though?

Reply:
Having it spawn a browser window to ask for authentication (which fails if you are authenticated) also doesn't feel like a good test. Most of the other commands also rely on talking to Netlify which feels problematic in a test suite.

The fact that the command exists and runs is not enough?

Reply:
Is there a command that needs a prior login to succeed? We could just run that, expect a failure and a specific message.

> The fact that the command exists and runs is not enough?

Generally help commands do not even load dependencies etc. So they don't give as much confidence as expecting an actual feature to fail.

Reply:
> Is there a command that needs a prior login to succeed? We could just run that, expect a failure and a specific message.

Great idea - thanks! I'll do some testing and update.